tasks/full_execution.yml

```diff
@@ -35,7 +35,7 @@
         ovirt_engine_setup_firewall_manager: null
         ovirt_engine_setup_answer_file_path: /root/ovirt-engine-answers
         ovirt_engine_setup_use_remote_answer_file: true
-        ovirt_engine_setup_update_all_packages: false
+        ovirt_engine_setup_update_all_packages: "{{ ovirt_engine_setup_update_all_packages | default(false) }}"
```

Comment:
maybe it will be better to add it to the doc in:
https://github.com/oVirt/ovirt-ansible-hosted-engine-setup#general-variables

Reply:
and it should match the ovirt_engine_setup_offline value because it's not going to work very well if it differs.
maybe worth an enhancement in ovirt-ansible-engine-setup to squash these two variables into a single one

Reply:
well, opened https://github.com/oVirt/ovirt-ansible-engine-setup/issues/74 to see what Martins think...